utils/invalid_input.js

```diff
@@ -0,0 +1,6 @@
+function sendInvalidInputMessage(res, message) {
+    res.throw(422, message);
+    return false;
```

Comment:
No need to return anything here. This line is unreachable because `res.throw` always throws.

Reply:
This simplifies quite a bit of logic both here and in the request validation checks :)